docker-amundsen-atlas.yml

```diff
@@ -14,7 +14,7 @@ services:
       image: amundsendev/amundsen-search:1.4.2
       container_name: amundsensearch
       ports:
-        - 5001:5000
+        - 5001:5001
```

Comment:
given the tag image is same as the pypi version, I think we don't need to change it now. Once we bump the pypi version, we could change that. Otherwise the docker file may not work.

My bad, initially I thought we need to change the port for every file.